playkit/src/main/java/com/kaltura/playkit/PlayerLoader.java

```diff
@@ -168,10 +166,9 @@ public void prepare(@NonNull final PKMediaConfig mediaConfig) {
             loadedPluginEntry.getValue().plugin.onUpdateMedia(mediaConfig);
         }
 
-        if (!isKavaImpressionFired && !PKDeviceCapabilities.isKalturaPlayerAvailable() &&
+        if (!PKDeviceCapabilities.isKalturaPlayerAvailable() &&
```

Comment:
@GouravSna 

I think it will be fired also in bg/fg not sure we want this please consult Ron

Reply:
It is working as expected in bg/fg